android/.gitignore

```diff
@@ -1,14 +1,21 @@
 .gradle
-local.properties
-.idea
 build/
 .DS_Store
 *.iml
 *.apk
 *.aar
 *.zip
-google-services.json
+googleservices.json
 
 .project
 .settings
 .classpath
+
+local.properties
+
+.idea/*
+!idea/codeStyles
```

Comment:
hm, is there a reason we need this now?

Reply:
This helps you to have same codestyle for all devs